bot/common.py

```diff
@@ -761,6 +779,13 @@ def pull_server_logs(args):
 
                     if file_path.endswith('.xml') and not 'tc_log' in file_path\
                             and b'Final Verdict:PASS' in file_bin.data:
+                        (test_name, timestamp) = parse_xml(file_path)
+                        print(test_name, timestamp)
+                        if test_name in last_xml[0]:
+                            if timestamp > last_xml[1]:
+                                os.remove(last_xml[0])
+                            else:
+                                continue
```

Comment:
```suggestion
                        (test_name, timestamp) = split_xml_filename(file_path)
                        if test_name in last_xml[0]:
                            if timestamp <= last_xml[1]:
                                continue
                            os.remove(last_xml[0])
                            
                                
```

Reply:
This looks much cleaner :) I updated the PR